index.html

```diff
@@ -55,6 +55,162 @@ <h1>I’m Mariane Glad to see you!</h1>
+                    <span class="sub-different-color bullet">•</span>
+                    <span class="sub-different-color">2015</span>
+                </div>
+                <p>
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+                <ul>
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javascript</li>
+                </ul>
+                <a href="#" class="project-details">See Project</a>
+            </div>
+            <div class="project">
+                <img src="images/project3.png" alt="project3 icon">
+                <p class="project-title">Tonic</p>
+                <div class="project-sub-header">
+                    <span>CANOPY</span>
+                    <span class="sub-different-color bullet">•</span>
+                    <span class="sub-different-color">Back End Dev</span>
+                    <span class="sub-different-color bullet">•</span>
+                    <span class="sub-different-color">2015</span>
+                </div>
+                <p>
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+                <ul>
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javascript</li>
+                </ul>
+                <a href="#" class="project-details">See Project</a>
+            </div>
+            <div class="project">
+                <img src="images/project4.png" alt="project4 icon">
+                <p class="project-title">Multi-Post Stories</p>
+                <div class="project-sub-header">
+                    <span>CANOPY</span>
+                    <span class="sub-different-color bullet">•</span>
+                    <span class="sub-different-color">Back End Dev</span>
+                    <span class="sub-different-color bullet">•</span>
+                    <span class="sub-different-color">2015</span>
+                </div>
+                <p>
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+                <ul>
+                    <li>html</li>
+                    <li>css</li>
+                    <li>javascript</li>
+                </ul>
+                <a href="#" class="project-details">See Project</a>
+            </div>
+        </div>
+
+        <div class="about">
+            <header class="header">
+                <p class="about-title">About Myself</p>
+                <p>
+                    Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+                </p>
+                <a href="#">LET’S CONNECT</a>
```

Comment:
- [ ] The LETS CONNECT section should be wrapped inside of the `<hx>` tags.

Reply:
Done.